src/viam/sdk/common/private/proto_utils.hpp

```diff
@@ -6,8 +6,7 @@
 
 #include <viam/api/common/v1/common.pb.h>
 
-namespace viam {
-namespace sdk {
+namespace viam::sdk::impl {
```

Comment:
Oops, sorry, I misled you with my comment. This syntax is c++17 only. You still need to do it with multiple lines. I was just trying to be clear that the `impl` namespace should be the innermost:

```
namespace viam {
namespace sdk {
namespace impl {
```

Reply:
oops -- switched back in 3def54d5